pycytominer/tests/test_cyto_utils/test_DeepProfiler_processing.py

```diff
@@ -26,29 +30,38 @@
 index_file = os.path.join(example_project_dir, "inputs", "metadata", "test_index.csv")
 
 output_folder = os.path.join(tmpdir, "DeepProfiler")
-
+os.makedirs(output_folder, exist_ok=True)
 
 # calculating the dataframe for each depth
-site_class = AggregateDeepProfiler(
+deep_data = DeepProfilerData(
     index_file=index_file,
     profile_dir=profile_dir,
+)
+
+single_cells_DP = SingleCellDeepProfiler(deep_data=deep_data)
+output_file = os.path.join(output_folder, "normalized.csv")
+single_cells = single_cells_DP.get_single_cells(output=True)
+single_cells_normalized = single_cells_DP.normalize_deep_single_cells(
```

Comment:
Can you add a test where you apply `normalize()` to `single_cells` as described in 43 using the exact same normalize logic used in `.normalize_deep_single_cells()` and `pd.DataFrame.assert_equal`?

It would be good to test the internal normalize logic has expected behavior as an external application.

Reply:
Unfortunately I had to do a bit of strangling with the `locations` to make the DP data compatible with `normalize()`. Let me know if the changes in https://github.com/cytomining/pycytominer/pull/210/commits/b75425114246786fdbe8bc42434ee174d01809c2 work for the pandas assertion.